### ISSUE TITLE
Creation of new stache site fails unable to find local grunt

### ISSUE THREAD

**Issue by Blackbaud-AaronDershem** [blackbaud-community/stache-cli#3]
When attempting to create a new stache site you receive _Fatal error: unable to find local grunt_.  I have tested this on Windows, Ubuntu and OS X with the latest version of the blackbaud-stache-cli package.   Using git bisect the last good version is 0.2.59.  Below is the output:

```
$ npm ls blackbaud-stache-cli -g
C:\Users\Blackbaud-AaronDershem\AppData\Roaming\npm
└── blackbaud-stache-cli@0.2.59


Blackbaud-AaronDershem@mymachine MINGW64 /c/github/blackbaud/test
$ stache new test.0.2.59
Running "new:test.0.2.59" (new) task

Running "copy:boilerplate" (copy) task
Created 22 directories, copied 40 files

Running "fixIgnore" task

Done, without errors.

$ npm install blackbaud-stache-cli -g
npm WARN deprecated lodash@0.9.2: Grunt needs your help! See https://github.com/gruntjs/grunt/issues/1403.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs version 3 and before will fail on newer node releases. Please                                                 update to graceful-fs@^4.0.0 as soon as possible.
C:\Users\Blackbaud-AaronDershem\AppData\Roaming\npm\stache -> C:\Users\Blackbaud-AaronDershem\AppData\Roaming\npm\node_modules\blackbaud-stache                                                -cli\bin\stache
blackbaud-stache-cli@0.3.9 C:\Users\Blackbaud-AaronDershem\AppData\Roaming\npm\node_modules\blackbaud-stache-cli
├── grunt-shell@1.2.1 (chalk@1.1.1)
├── grunt-contrib-copy@0.8.2 (file-sync-cmp@0.1.1, chalk@1.1.1)
├── grunt-cli@0.1.13 (resolve@0.3.1, nopt@1.0.10, findup-sync@0.1.3)
└── grunt@0.4.5 (eventemitter2@0.4.14, dateformat@1.0.2-1.2.3, which@1.0.9, async@0.1.22, getobject@0.1.0, colors                                                @0.6.2, lodash@0.9.2, rimraf@2.2.8, hooker@0.2.3, grunt-legacy-util@0.2.0, exit@0.1.2, nopt@1.0.10, coffee-script                                                @1.3.3, iconv-lite@0.2.11, underscore.string@2.2.1, minimatch@0.2.14, glob@3.1.21, grunt-legacy-log@0.1.3, findup                                                -sync@0.1.3, js-yaml@2.0.5)

Blackbaud-AaronDershem@mymachine MINGW64 /c/github/blackbaud/test
$ npm ls blackbaud-stache-cli -g
C:\Users\Blackbaud-AaronDershem\AppData\Roaming\npm
└── blackbaud-stache-cli@0.3.9


Blackbaud-AaronDershem@mymachine MINGW64 /c/github/blackbaud/test
$ stache new test.0.3.9
grunt-cli: The grunt command line interface (v1.1.0)

Fatal error: Unable to find local grunt.

If you're seeing this message, grunt hasn't been installed locally to
your project. For more information about installing and configuring grunt,
please see the Getting Started guide:

http://gruntjs.com/getting-started

```

**Comment by Blackbaud-BobbyEarl** [comment]
Thanks @Blackbaud-AaronDershem.  Tagging @Blackbaud-TripOtt and @Blackbaud-BenJacobs as they too saw the behavior.  Should be able to do a release tomorrow with a fix.

**Comment by Blackbaud-SteveBrush** [comment]
This has been fixed with 0.3.10! Thanks for pointing this out, @Blackbaud-AaronDershem.
